src/plib/resources/templates/plesk.com.web.json

```diff
@@ -10,16 +10,47 @@
     "variableDescription": "ip is the IP address of the server.",
     "records": [
         {
+            "groupId": "WebService",
             "type": "A",
             "host": "@",
             "pointsTo": "%ip%",
             "ttl": "600"
         },
         {
+            "groupId": "WebService",
             "type": "CNAME",
             "host": "www",
             "pointsTo": "@",
             "ttl": "600"
+        },
+        {
+            "groupId": "MailService",
+            "type": "MX",
+            "pointsTo": "%mxhost%",
+            "host": "@",
+            "priority": "%mxpriority%",
```

Comment:
propose to keep the template more precise and put the values instead of variables.

Reply:
What is the point if we use hardcoded values instead of variable we've taken from zone?